airbyte-integrations/connectors/source-mysql/src/main/java/io/airbyte/integrations/source/mysql/DebeziumRecordPublisher.java

```diff
@@ -152,6 +152,7 @@ protected Properties getDebeziumProperties(JsonNode config,
     // https://debezium.io/documentation/reference/connectors/mysql.html#mysql-boolean-values
     props.setProperty("converters", "boolean");
     props.setProperty("boolean.type", "io.debezium.connector.mysql.converters.TinyIntOneToBooleanConverter");
+    props.setProperty("decimal.handling.mode", "string");
```

Comment:
can you add a comment explaining: 
1. why we set this property 
2. whether it results in a loss of precision

Reply:
Done, thanks